dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -157,7 +157,6 @@ FROM
     ('fx-function-x', 'ethereum', 'FX', 0x8c15ef5b4b21951d50e53e4fbda8298ffad25057, 18),
     ('fxc-flexacoin', 'ethereum', 'FXC', 0x4a57e687b9126435a9b19e4a802113e266adebde, 18),
     ('fxs-frax-share', 'ethereum', 'FXS', 0x3432b6a60d23ca0dfca7761b7ab56459d9c964d0, 18),
-    ('gala-gala', 'ethereum', 'GALA', 0x15d4c048f83bd7e37d49ea4c83a07267ec4203da, 8),
```

Comment:
Do you think we can fix the token address to reuse this price feed?

Reply:
good question. can we update address to the below?
`0xd1d2Eb1B1e90B638588728b4130137D262C87cae`

coinpaprika has a price that matches what i see on etherscan for the new address:
https://coinpaprika.com/coin/gala-gala/

would this work?

Reply:
ah, nevermind, i see in your PR description now a query that shows this token is in prices twice already with both addresses. we are simply removing the old one.